specs/testing/SkillsMatrixTesting.md

```diff
@@ -0,0 +1,104 @@
+| Python          |     7          |   8         |   9     |       19          |       20        |    21   |
+| C# Net Core 2.1 |      10         |    11        |  12      |      22           |      23         |  24     |
+
+1. [Unchanged] C# Bot acting as consumer, calling a C# Skill
+2. [Unchanged] C# Bot acting as consumer, calling a JS Skill
+3. [Unchanged] C# Bot acting as consumer, calling a Pyton Skill
+4. [Unchanged] JS Bot acting as consumer, calling a C# Skill
+5. [Unchanged] JS Bot acting as consumer, calling a JS Skill
+6. [Unchanged] JS Bot acting as consumer, calling a Python Skill
+7. [Unchanged] Python Bot acting as consumer, calling a C# Skill
+8. [Unchanged] Python Bot acting as consumer, calling a JS Skill
+9. [Unchanged] Python Bot acting as consumer, calling a Python Skill
+10. [New] C# Net Core 2.1 Bot acting as a consumer, calling a C# .Net core 3.1 Skill 
+11. [New] C# Net Core 2.1 Bot acting as a consumer, calling a JS Skill 
+12. [New] C# Net Core 2.1 Bot acting as a consumer, calling a Python Skill 
+13. [New] C# Net Core 3.1 Bot acting as a consumer, calling a .Net Core 2.1 Skill 
+14. [New] C# Net Core 3.1 Bot acting as a consumer, calling a v3 JS Skill 
+15. [New] C# Net Core 3.1 Bot acting as a consumer, calling a v3 C# Skill
+16. [New] JS Bot acting as consumer, calling a C# .Net Core 2.1 Skill
+17. [New] JS Bot acting as consumer, calling a v3 JS Skill
+18. [New] JS Bot acting as consumer, calling a v3 C# Skill
+19. [New] Python Bot acting as consumer, calling a C# Net Core 2.1 Skill
+20. [New] Python Bot acting as consumer, calling a v3 JS  Skill
+21. [New] Python Bot acting as consumer, calling a v3 C#  Skill
+22. [New] C# Net Core 2.1 acting as consumer, calling a C# .NEt Core 2.1 Skill
+23. [New] C# Net Core 2.1 acting as consumer, calling a v3 JS Skill
+24. [New] C# Net Core 2.1 acting as consumer, calling a v4 C# Skill
+
+
+# Tests
+* **Crawl** The only test run is an Echo + end utterance to trigger an EndOfConversation Activity. The Consumer sends a Message containing a GUID to the skill, and the skill encho that back. The skill then sends end the conversation. 
+* **Walk** Scenario for Events and Invoke + InvokeResponse are added. 
+* **Run** the test matrix grows to include C# .Net Core 2.1 (as a Skill Host and Skill), and v3 JS (as a Skill) and v3 C# (as as Skill)
+
+# Bots we need
+3 Bots (1 in C#, one in JS, one in Python). The Bot picks up "Skill" or "Consumer" from the ENV variables, which are set as part of the bot's deployment. 
+
+The bots that may server as a starting point for development are here:
+[C#](https://github.com/microsoft/BotBuilder-Samples/tree/master/samples/csharp_dotnetcore/80.skills-simple-bot-to-bot)
+[JS](https://github.com/microsoft/BotBuilder-Samples/tree/master/samples/javascript_nodejs/80.skills-simple-bot-to-bot)
+[Python] (https://github.com/microsoft/BotBuilder-Samples/tree/master/samples/python/80.skills-simple-bot-to-bot)
```

Comment:
Extra space after "Python]" (the link doesn't render right)
![image](https://user-images.githubusercontent.com/12264946/73894338-2ee8a600-484a-11ea-8fca-064c20ee1df8.png)

Reply:
Fixed